src/SFA.DAS.Forecasting.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -20,6 +21,44 @@ public static class HtmlHelperExtensions
             }
 
             return new MvcHtmlString("error");
-        }   
+        }
+
+
+        public static MvcHtmlString SetZenDeskLabels(this HtmlHelper html, params string[] labels)
+        {
+            var apiCallString =
+                "<script type=\"text/javascript\">zE('webWidget', 'helpCenter:setSuggestions', { labels: [";
+
+            var first = true;
+            foreach (var label in labels)
+            {
+                if (!string.IsNullOrEmpty(label))
+                {
+                    if (!first) apiCallString += ",";
+                    first = false;
+
+                    apiCallString += $"'{ EscapeApostrophes(label) }'";
+                }
+            }
+
+            apiCallString += "] });</script>";
+
+            return MvcHtmlString.Create(apiCallString);
+        }
+
+        private static string EscapeApostrophes(string input)
+        {
+            return input.Replace("'", @"\'");
+        }
+
+        public static string GetZenDeskSnippetKey(this HtmlHelper html)
+        {
+            return ConfigurationManager.AppSettings["ZenDeskSnippetKey"];
+        }
+
+        public static string GetZenDeskSnippetSectionId(this HtmlHelper html)
+        {
+            return ConfigurationManager.AppSettings["ZenDeskSectionId"];            
+        }
```

Comment:
Are there any tests covering this? Does it matter if these aren't set?

Reply:
was thinking initially to add Test for this. After i have the config changes https://github.com/SkillsFundingAgency/das-forecasting-tool/pull/244 , we know the variable been added to the pipeline